src/components/ebay-signal/index.marko

```diff
@@ -0,0 +1,5 @@
+$ var type = input.type;
```

Comment:
We should have a default value
Also I think we generally don't like `type`. Maybe `status` is better?